spec/orders_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'orders'
+# require 'sms'
+
+describe Orders do
+
+  let(:menu) { double :menu, :menu_items => { :pie => 4.5, :chips => 2.50 } }
+  let(:mockbill) { double :mockbill, :bill => 14 }
+  let(:bill_klass) { double :bill_klass, :new => mockbill }
+
+  it 'chooses the menu' do
+    expect(subject.choose_menu(menu)).to eq menu.menu_items
+  end
+
+  it 'responds to 2 arguments' do
+    expect(subject).to respond_to(:cust_order).with(2).arguments
```

Comment:
This style of `respond_to` test are a great way to get started with the TDD process, but one you write more in depth tests they become redundant and can be deleted.